pyglmnet/pyglmnet.py

```diff
@@ -676,6 +676,9 @@ def fit(self, X, y):
         self.beta0_ = beta[0]
         self.beta_ = beta[1:]
         self.ynull_ = np.mean(y)
+
+        # Update loss
+        self._loss = L
```

Comment:
@pavanramkumar I profiled the code and it seems computing the loss function takes up 1/4th of the time. You could simply use the norm of the gradient for stopping criteria and make it more efficient.